service-front/assets/js/moj/moj.modules/moj.user-timeout.js

```diff
@@ -1,105 +0,0 @@
-// User Timeout module for LPA
-// Dependencies: moj, _, jQuery
-
-(function () {
-  'use strict';
-
-  moj.Modules.UserTimeout = {
-    timeoutDuration: 1000 * 60 * 55, // ms * s * m = 55 minutes
-    gracePeriod: 1000 * 60 * 5, // ms * s * m = 5 minutes
-    timeout: null,
-
-    init: function () {
-      _.bindAll(this, 'warning');
-
-      var startTime = this.time();
-      // check to see if user is logged in
-
-      if (false) {
-
-        $.ajax({
-          url: '/user/is-logged-in',
-          dataType: 'json',
-          timeout: 10000,
-          cache: false,
-          context: this,
-          success: function (response) {
-            if (response.isLoggedIn) {
-              // if logged in, start setTimeout to alert them when timeout is imminent
-              this.timeout = setTimeout(this.warning, this.timeoutDuration - (this.time() - startTime));
-            }
-          }
-        });
-
-      }
-    },
-
-    warning: function () {
-      var html = '',
-        self = this;
-
-      // TODO: get this HTML out of the JS into a template
-      // TODO: get someone to look at this copy, I made it up. Clive.
-      html += '<div class="timeout-body">';
-      html += '<h1>Session timeout</h1>';
-      html += '<p>You have been inactive for 55 minutes and will be automatically logged out in 5 minutes.</p>';
```

Comment:
We don't ever see this hard-coded text in the app, which suggests it is unused. We also have a cypress test for the session timeout dialog box which passes after this code is removed.